api/src/main/java/io/opentelemetry/trace/Span.java

```diff
@@ -304,6 +324,15 @@ default void setAttribute(AttributeKey<Long> key, int value) {
    */
   boolean isRecording();
 
+  /**
+   * Returns whether this {@link Span} is valid.
+   *
+   * @see Span#getInvalid()
+   */
+  default boolean isValid() {
+    return getContext().isValid();
+  }
```

Comment:
I think this change is unrelated with the PR description and purpose, consider to not add "secondary" changes to a PR.

Reply:
Sure makes sense